pubspec.yaml

```diff
@@ -0,0 +1,6 @@
+name: wCommon
+description: Dart bindings for wf-common
+dependencies:
+dev_dependencies:
+  unittest: '>=0.11.0 <0.11.6'
+  mockito: '>=0.8.2'
```

Comment:
We can make this a specific version as well. It is < 1 which means it's unstable.